src/transformers/models/code_llama/tokenization_code_llama_fast.py

```diff
@@ -264,6 +264,7 @@ def set_infilling_processor(self, reset, suffix_first=False, add_special_tokens=
                 ]
             )
             self.update_post_processor()
+            return
```

Comment:
This method would really benefit from a docstring - it's not clear to me what's supposed to be happening and what `reset` is supposed to do

Reply:
Indeed!